packages/model-viewer/src/three-components/Renderer.ts

```diff
@@ -324,7 +325,7 @@ export class Renderer extends EventDispatcher {
         scene.scaleStep = 0;
         this.rescaleCanvas(scene);
       } else {
-        return false;
+        return scene.dirtyRender; // this is usually false, unless specified otherwise
```

Comment:
What about skipping this change and giving a `beforeRender()` callback for the effect that's called in `$tick`? That's how we call `queueRender` in e.g. our animationMixer. Might be helpful in case a custom effect takes input of some kind.

Reply:
Interesting, I didn't see that. Sounds like a better solution to me